site/en/blog/media-updates-in-chrome-69/index.md

```diff
@@ -0,0 +1,289 @@
+        contentType: 'audio/mp4; codecs="mp4a.40.2"',
+        encryptionScheme: 'cbcs'
+      },
+    ],
+    initDataTypes: ['keyids']
+  }]);
+```
+
+[Intent to Implement](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/lMUKOaohUTY) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5184416120832000) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=838416)
+
+## EME: HDCP policy check {: #hdcp}
+
+Nowadays [HDCP] is a common policy requirement for streaming high resolutions
+of [protected content]. And web developers who want to enforce an HDCP policy
+must either wait for the license exchange to complete or start streaming
+content at a low resolution. This, is a sad situation that the [HDCP Policy
+Check API] aims to solve.
+
+This proposed API allows web developers to query whether a certain HDCP policy
+can be enforced so that playback can be started at the optimum resolution for
+the best user experience. It consists of a simple method to query the status of
+a hypothetical key associated with an HDCP policy, without the need to create a
+`MediaKeySession` or fetch a real license. It does not require `MediaKeys` to be
+attached to any audio or video elements either.
+
+The HDCP Policy Check API works simply by calling
+`mediaKeys.getStatusForPolicy()` with an object that has a `minHdcpVersion` key
+and a valid value. If HDCP is available at the specified version, the returned
+promise resolves with a `MediaKeyStatus` of `'usable'`. Otherwise, the promise
+resolves with [other error values] of `MediaKeyStatus` such as
+`'output-restricted'` or `'output-downscaled'`. If the key system does not
+support HDCP Policy Check at all (e.g. Clear Key System), the promise rejects.
+
+In a nutshell, here's how the API works for now. Check out the [official sample]
+to try out all versions of HDCP.
+
+```js
+const config = [{
+  videoCapabilities: [{
+    contentType: 'video/webm; codecs="vp09.00.10.08"',
+    robustness: 'SW_SECURE_DECODE' // Widevine L3
+  }]
+}];
+
+navigator.requestMediaKeySystemAccess('com.widevine.alpha', config)
+.then(mediaKeySystemAccess => mediaKeySystemAccess.createMediaKeys())
+.then(mediaKeys => {
+
+  // Get status for HDCP 2.2
+  return mediaKeys.getStatusForPolicy({ minHdcpVersion: '2.2' })
+  .then(status => {
+    if (status !== 'usable')
+      return Promise.reject(status);
+
+    console.log('HDCP 2.2 can be enforced.');
+    // TODO: Fetch high resolution protected content...
+  });
+})
+.catch(error => {
+  // TODO: Fallback to fetch license or stream low-resolution content...
+});
+```
+
+### Available for origin trials
+
+To get feedback from web developers, we've previously added the HDCP Policy
+Check API feature in Chrome 69 for Desktop (Chrome OS, Linux, Mac, and Windows).
+
+The trial successfully ended in November 2018.
+
+[Intent to Experiment](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/ITzZ_yx4bF8) &#124;
```

Comment:
What is the trailing &#124; doing here? It's the pipe symbol. Seems like maybe a mistake in import that shows up in a few places?

Reply:
It's not a mistake. See https://developers.google.com/web/updates/2018/08/chrome-69-media-updates#:~:text=Intent%20to%20Implement%20%7C%20Chromestatus%20Tracker%20%7C%20Chromium%20Bug